barretenberg/cpp/scripts/ci_benchmark_ivc_flows.sh

```diff
@@ -104,67 +105,23 @@ export -f verify_ivc_flow run_bb_cli_bench
 
 chonk_flow $1 $2
 
-# Upload benchmark breakdown (op counts and timings) to gh-pages if running in CI and it's a native build
-if [[ "${CI:-}" == "1" ]] && [[ "$1" == "native" ]]; then
-  echo_header "Uploading Barretenberg benchmark breakdowns to gh-pages"
+# Upload benchmark breakdown (op counts and timings) to disk if running in CI and it's a native build
+if [[ "${CI:-}" == "1" ]] && [[ "$1" == "native" ]] && [[ "${CI_ENABLE_DISK_LOGS:-0}" == "1" ]]; then
+  echo_header "Uploading Barretenberg benchmark breakdowns"
 
   flow_name="$(basename $2)"
   benchmark_breakdown_file="bench-out/app-proving/$flow_name/native/benchmark_breakdown.json"
 
   if [[ -f "$benchmark_breakdown_file" ]]; then
     current_sha=$(git rev-parse HEAD)
 
-    # CI needs to authenticate from GITHUB_TOKEN.
-    gh auth setup-git &>/dev/null || true
-
-    # Shallow clone gh-pages branch with retry logic
-    rm -rf gh-pages-repo
-    for clone_attempt in 1 2 3; do
-      if git clone --depth 1 --branch gh-pages "$(git config --get remote.origin.url)" gh-pages-repo; then
-        break
-      else
-        echo "Clone failed, retrying (attempt $clone_attempt/3)..."
-        rm -rf gh-pages-repo
-        if [[ $clone_attempt -eq 3 ]]; then
-          echo "Failed to clone gh-pages after 3 attempts"
-          exit 1
-        fi
-        sleep 2
-      fi
-    done
-
-    # Create target directory for this flow
-    mkdir -p "gh-pages-repo/bench/barretenberg-breakdowns/${flow_name}"
-
-    # Copy benchmark breakdown JSON (contains op counts and timing data) with commit SHA as filename
-    cp "$benchmark_breakdown_file" "gh-pages-repo/bench/barretenberg-breakdowns/${flow_name}/${current_sha}.json"
-
-    # Commit and push with retry on race condition
-    cd gh-pages-repo
-    git config user.name "Aztec Bot"
-    git config user.email "bot@aztec.network"
-    git add bench/barretenberg-breakdowns/
-    if ! git diff --staged --quiet; then
-      git commit -m "Update Barretenberg benchmark breakdowns (op counts and timings) for ${current_sha}"
-
-      # Retry push up to 3 times in case of race conditions
-      for attempt in 1 2 3; do
-        if git push; then
-          break
-        elif [[ $attempt -lt 3 ]]; then
-          echo "Push failed, retrying after fetch and rebase (attempt $attempt/3)..."
-          git fetch origin gh-pages
-          git rebase origin/gh-pages
-        else
-          echo "Push failed after 3 attempts"
-          exit 1
-        fi
-      done
-    fi
-    cd ..
-
-    # Clean up the shallow clone
-    rm -rf gh-pages-repo
+    # Create cache key: bench/bb-breakdown/<flow_name>/<sha>
+    cache_key="bench/bb-breakdown/${flow_name}/${current_sha}"
+
```

Comment:
can we read this cache key from ci.aztec-labs.com/XXX? not a hard requirement, but could be nice